### ISSUE TITLE
Watch for `KycOnboarding .Onboarded`, `CouponOnboarding. CouponRedeemed`

### ISSUE THREAD

**Issue by jtrein** [openlawteam/tribute-discord-service#43]
**Problem**

There is no notification for whom has onboarded via `KycOnboarding` or `CouponOnboarding` for DAOs such as Neon, RPD, unless one checks the Coupon Manager UI / database, or refreshes the Dapps' Members pages. When we need to know 

**Feature Request**

Via Discord DM with @jdville03 and @sophiacodes :

> Cool yeah that would be helpful to watch for both. We also using CouponOnboarding in Muse0 (as a fallback when the TributeNFT adapter and NFTExtension can't be used for various reasons including if the NFT is a ERC1155).

> We're using both onboarding methods for the DAOs so one is not really more vital than the other. Generally the pattern for Neon and RPD is that KycOnboarding is used to onboard new investors and then CouponOnboarding is used to onboard existing members (since those DAOs were already formed before we built the Dapps for them). CouponOnboarding has also been used to onboard folks in unique situations (e.g., like how we've been onboarding Aaron, Pri, Arno into these DAOs as "admin" members so they can view and put up proposals; we also had to onboard Felipe to RPD so he could handle the proposals required to update some configurations).

**Comment by jtrein** [comment]
Would be good to handle a case when an admin is added. @jdville03 Admins have `1` share, typically?

**Comment by jdville03** [comment]
Yes that's correct that when we onboard one of our team members just so they can perform "admin" type actions on behalf of the other actual DAO members we just issue `1` unit. The only other member address that usually only has `1` unit is the DAO creator (assuming that address doesn't later get more units as an actual DAO member).
